projects/PM_hash_navigation/main.cc

```diff
@@ -95,14 +97,14 @@ main(int argc, char **argv)
     // ----- camera loop ---------------
     unsigned int frame = 0;
 
+    cv::Mat rotatedView, resizedRotatedView, image, resized;
+    cv::Mat outputImage(unwrapRes, CV_8UC3);
```

Comment:
Declare these outside the body of the loop. If you do it inside the loop then for every iteration your program will have to walk the heap to find space to allocate memory, allocate it and then walk it again to free the memory :wink:, so this could be part of the reason things are running slowly for you!